compiler/src/iree/compiler/Dialect/Util/Conversion/ConversionPatterns.cpp

```diff
@@ -169,6 +169,30 @@ struct ConvertReturnOp : public OpConversionPattern<IREE::Util::ReturnOp> {
   }
 };
 
+struct ConvertUnreachableOp
+    : public OpConversionPattern<IREE::Util::UnreachableOp> {
+  using OpConversionPattern::OpConversionPattern;
+  LogicalResult
+  matchAndRewrite(IREE::Util::UnreachableOp op, OpAdaptor adaptor,
+                  ConversionPatternRewriter &rewriter) const override {
+    rewriter.replaceOpWithNewOp<IREE::Util::UnreachableOp>(op,
+                                                           op.getMessageAttr());
+    return success();
+  }
+};
+
+struct ConvertSCFUnreachableOp
+    : public OpConversionPattern<IREE::Util::SCFUnreachableOp> {
+  using OpConversionPattern::OpConversionPattern;
+  LogicalResult
+  matchAndRewrite(IREE::Util::SCFUnreachableOp op, OpAdaptor adaptor,
+                  ConversionPatternRewriter &rewriter) const override {
+    rewriter.replaceOpWithNewOp<IREE::Util::SCFUnreachableOp>(
+        op, op.getMessageAttr());
+    return success();
+  }
+};
+
```

Comment:
They are identical to the patterns in the other file: Patterns.cpp. Why do we need both?